Sources/SwiftGodot/SwiftGodot.docc/Tutorials/Your First Extension.tutorial

```diff
@@ -65,7 +65,7 @@
         
         @Steps {
             @Step {
-                Create a new Swift file called **PlayerController** in the SimpleRunnerDriver package and import the 
+                Create a new Swift file called **PlayerController.swift** in the `Sources/SimpleRunnerDriver` directory and import the 
```

Comment:
The tutorial's text clearly states that you're creating a Swift file, I feel the addition of the file extension here is a little redundant?

Reply:
makes it copy pastable, probably should be in backticks

Reply:
I don’t think that, in this instance, it is necessary to make it copy/paste-able. While it may be convenient, it doesn’t contribute anything meaningful to the tutorial or the documentation itself.